README-dev.md

```diff
@@ -31,7 +31,7 @@ The short version:
 
 - Invoke `make macos-setup`
   - If this is your first time using OCaml, be sure to run `eval $(opam config env)`
-- Invoke `rustup default 1.45.2`
+- Invoke `rustup default 1.52.1`
```

Comment:
you shouldn't need to do this as it's already specified in the `rust-toolchain.toml` file

Reply:
There isn't a `rust-toolchain.toml` in the `marlin_plonk_bindings/stubs` library, so for now this is correct. I'll check in that file and update here.